### PR TITLE
Component vertical and horizontal alignment

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Component vertical and horizontal alignment

Allow the ability to configure text alignment

## Motivation and Context
Playlist Epic

## How Has This Been Tested?
Tested locally. Initial PR will update accordingly for release.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

File: demo/src/rise-text-dev.html [hunk 1]
```diff
@@ -29,13 +29,23 @@
...
+  </style>
 </head>
 
 <body>
-
-  <rise-text value="Hello
-   World!" fontsize="100" multiline="true">
-  </rise-text>
+  <div class="text-container">
+    <rise-text value="Hello World!" fontsize="25" verticalalign="middle" horizontalalign"center" textalign="center"></rise-text>
```
Comment (olegrise) [inline demo/src/rise-text-dev.html:43]: @alex-deaconu should it be `horizontalalign="center"`

Comment (alex-deaconu) [reply demo/src/rise-text-dev.html:43]: @olegrise good catch, will fix, thanks!

**Comment by alex-deaconu** [comment]
Will bump version after https://github.com/Rise-Vision/rise-text/pull/60 is merged.

@ezequielc @stulees @olegrise please review. Thanks!

**Review by olegrise — APPROVED** [review]

**Review by stulees — APPROVED** [review]
@alex-deaconu Looks good to me

**Pull request merged**